app/webpacker/requests/NotSignedIn.js

```diff
@@ -2,7 +2,7 @@ import React, { Fragment, useState } from 'react';
 
 import Alert from 'react-bootstrap/Alert';
 
-import I18n from 'i18n-for-js';
+import I18n from 'i18n-for-js/index.js.erb';
```

Comment:
Comment to not forget this: I should probably factorize all ruby-related stuff in a single erb-loaded file.